sputnikdao2/src/lib.rs

```diff
@@ -372,4 +373,26 @@ mod tests {
             },
         });
     }
+
+    #[test]
+    fn test_update_policy() {
```

Comment:
This is a pretty sensible implementation that can lead to nasty hidden bugs. I would like to see more tests, in particular:

- updating `default_vote_policy`
- updating `proposal_bond` or `bounty_bond` which are U128
- updating `bounty_forgiveness_period` or `proposal_period` which are U64
- updating an entire role, not just the name. So the path is `roles/0` and `new_value` is an  `RolePermission` object serialized.